.github/workflows/deploy.yml

```
@@ -78,26 +84,51 @@ jobs:
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v4
+      - name: Health Check: Verify site is live
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)